kafka/Dockerfile

```diff
@@ -26,3 +26,10 @@ RUN tar -xvf ${package} && \
     rm ${package}
 
 COPY --from=build /home/gradle/src/build/libs /kafka/connect/ed-fi-kafka-connect-transforms
+
+# Kafka connect SMT to expand JSON string: https://github.com/RedHatInsights/expandjsonsmt
+ARG expandjsonsmt=kafka-connect-smt-expandjsonsmt-0.0.7.tar.gz
+ADD --chown=kafka --chmod=600 https://github.com/RedHatInsights/expandjsonsmt/releases/download/0.0.7/${expandjsonsmt} \
+    /kafka/connect/
+    RUN tar -xvf ${expandjsonsmt} && \
+    rm ${expandjsonsmt}
```

Comment:
It will be best if we can check the SHA256 hash value on this, similar to lines 21-23.

Reply:
Yep, just hadn't gotten to that yet 🙂